### PR TITLE
Remove image size limit

### PR DESCRIPTION
**Pull request by Lin-Rexter**
Increase image resolution to 1024x1024

### CODE REVIEW

File: src/ImageGen.py [hunk 1]
```diff
@@ -37,14 +37,14 @@ def get_images(self, prompt: str) -> list:
...
-        url = f"{BING_URL}/images/create?q={url_encoded_prompt}&rt=4&FORM=GENCRE"
+        # https://www.bing.com/images/create?q=<PROMPT>&rt=3&FORM=GENCRE
+        url = f"{BING_URL}/images/create?q={url_encoded_prompt}&rt=3&FORM=GENCRE"
         response = self.session.post(url, allow_redirects=False)
         if response.status_code != 302:
             print(f"ERROR: {response.text}")
             raise Exception("Redirect failed")
         # Get redirect URL
-        redirect_url = response.headers["Location"]
+        redirect_url = response.headers["Location"].replace("&nfy=1", "")
```
Comment (acheong08) [inline src/ImageGen.py:47]: And what is this for?

File: src/ImageGen.py [hunk 2]
```diff
@@ -64,8 +64,10 @@ def get_images(self, prompt: str) -> list:
 
         # Use regex to search for src=""
         image_links = regex.findall(r'src="([^"]+)"', response.text)
+        # Remove size limit
+        normal_image_links = [link.split("?w=")[0] for link in image_links]
```
Comment (acheong08) [inline src/ImageGen.py:68]: Can we make this optional?

Comment (Lin-Rexter) [reply src/ImageGen.py:68]: Yes, we can make this an option, it's a good idea.

File: src/ImageGen.py [hunk 3]
```diff
@@ -37,14 +37,14 @@ def get_images(self, prompt: str) -> list:
         """
         print("Sending request...")
         url_encoded_prompt = urllib.parse.quote(prompt)
-        # https://www.bing.com/images/create?q=<PROMPT>&rt=4&FORM=GENCRE
-        url = f"{BING_URL}/images/create?q={url_encoded_prompt}&rt=4&FORM=GENCRE"
+        # https://www.bing.com/images/create?q=<PROMPT>&rt=3&FORM=GENCRE
+        url = f"{BING_URL}/images/create?q={url_encoded_prompt}&rt=3&FORM=GENCRE"
```
[the remark this answers is not in the document]
Comment (Lin-Rexter) [reply src/ImageGen.py:41]: Sorry, I'm not sure, I found it by accident during the adjustment process.

File: src/ImageGen.py (hunk 1 above)
Comment (Lin-Rexter) [reply src/ImageGen.py:47]: The same as above.

**Pull request merged**